recipes/ncbi-cxx-toolkit-public/config.yml

```
@@ -0,0 +1,3 @@
+versions:
+  "26.0.1":
+    folder: 26.0
```

```suggestion
    folder: "26.0"
```

We typically use `all` for the folder name until versions start to diverge too much from each other.